nextcord/embeds.py

```diff
@@ -215,6 +222,9 @@ def __init__(
         if timestamp:
             self.timestamp = timestamp
 
+        self._local_image: Optional[File] = None
```

Comment:
type issue here is caused by
https://github.com/nextcord/nextcord/pull/653/files#diff-223279fd9bce07b3afa63f9f5d80fdaf27d8bc90b6fa894cf4fdaeca7037a49aR484